### ISSUE TITLE
Accessibility auto-fill cannot be used with accessibility shortcuts

### ISSUE THREAD

**Issue by lucasmz-dev** [bitwarden/android#4224]
### Steps To Reproduce

1. Enable accessibility auto-fill on Settings > Auto-fill
2. Enable an accessibility shortcut for it, such as holding both volume keys to toggle it
3. Leave Bitwarden and use the volume keys to disable and enable them
4. Come back to Bitwarden and check if accessibility auto-fill works

### Expected Result

Accessibility auto-fill should work despite being toggled using accessibility shortcuts.

### Actual Result

Accessibility auto-fill breaks when accessibility shortcuts are used, and believes it is off when it is in fact on.

### Additional Context

This didn't happen on the xamarin app

### Build Version

2024.10.2 - F-Droid variant from BW repo

### What server are you connecting to?

Self-host

### Self-host Server Version

vaultwarden

### Environment Details

- Device: Moto G52 "rhode"
- Android: 14
- OS: CalyxOS 5.11.2

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by lucasmz-dev** [comment]
I end up using it like this for security reasons, accessibility is naturally an invasive permission and it only becomes useful in certain situations, where I then end up using the volume keys to manually do it.

**Comment by BBGhub** [comment]
Hi there!

Thank you for your report, it seems like it is a duplicate of this one https://github.com/bitwarden/android/issues/3595

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time in there - our engineering team will be happy to review these.

Vaultwarden users, see this FAQ: https://bitwarden.com/help/hosting-faqs/#q-do-bitwarden-client-apps-support-non-official-servers

This issue will now be closed.

Thanks!

**Comment by lucasmz-dev** [comment]
Hey I don't think this constitutes a duplicate.

**Comment by fessmm** [comment]
@BBGhub how is that a duplicate? pls explain

**Comment by differsthecat** [comment]
Hello, I am on the engineering team at Bitwarden. 

We looked at this and cannot reproduce this issue with the latest release. We have made a number of bug fixes that may have addressed this as a side effect; @lucasmz-dev can you try the latest release (2024.11.3) and see if it has been resolved?

I have reopened this until we can confirm. If you are still seeing it, we will continue investigating in engineering.

**Comment by lucasmz-dev** [comment]
Nice one fixing them other bugs! Unfortunately I can still reproduce this on the latest version.

**Comment by differsthecat** [comment]
@lucasmz-dev Ah, that's unfortunate! We will keep investigating on our end, thank you.

**Comment by Bondog83** [comment]
I'm having same issue on galaxy S24

**Comment by Bondog83** [comment]
Still occurring. When is this going to be resolved?

**Comment by Bondog83** [comment]
> Hello, I am on the engineering team at Bitwarden.
> 
> We looked at this and cannot reproduce this issue with the latest release. We have made a number of bug fixes that may have addressed this as a side effect; @lucasmz-dev can you try the latest release (2024.11.3) and see if it has been resolved?
> 
> I have reopened this until we can confirm. If you are still seeing it, we will continue investigating in engineering. Thank you!

This most up to date version on the play store is 2024.10.0.

**Comment by lucasmz-dev** [comment]
@Bondog83 isn't that still the non-native app?

**Comment by Bondog83** [comment]
> @Bondog83 isn't that still the non-native app?

Not sure, guess Bitwarden refuses to update the app in the Play Store

**Comment by aj-rosado** [comment]
We have made some changes on https://github.com/bitwarden/android/pull/4864 and https://github.com/bitwarden/android/pull/4756 that should fix this on next release!

Closing it for now, if the issues persists after next release let us know.

Thank you for your patience.
